src/styles/blocks/catalog.scss

```diff
@@ -18,6 +20,14 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: transform $card-transition-duration,
+      color $card-transition-duration;
+
+    &:hover {
+      transform: scale(1.2);
+      color: #34568b;
```

Comment:
If color values are used in several places, makes sense to create variables for them, as well. Check it

Reply:
Not fixed

Reply:
> I don't see any changes from last review, make sure you made git commit + git push + deploy

We just had to wait a little bit